cms/static/sass/elements/_header.scss

```diff
@@ -118,7 +118,6 @@
       display: block;
 
       img {
-        width: 100%;
```

Comment:
I had to make this change because otherwise, the generic venn-diagram logo was scaled up to fill the available horizontal space, which caused it to take up too much vertical space. I tested this change with both the venn-diagram logo and the edX Studio logo, and they both look good.